DATA/MISSIONS/npcships.ini

```diff
@@ -120,656 +390,761 @@ npc_class = lawful, class_cruiser, d1, d2, d3, d4, d5, d6, d7, d8, d9, d10, d11,
 ;; ZIVIL Starflyer
 
 [NPCShipArch]
-nickname = npc_co_ge_fighter_d1-2
+nickname = npc_co_ge_fighter_d1_a
+;;nickname = npc_co_ge_fighter_d1-2
 level = d1
-loadout = co_ge_fighter_loadout_d1-2
+loadout = co_ge_fighter_loadout_d1_a
 ship_archetype = cv_ship_fighter_verylight01_npc
 state_graph = FIGHTER
 pilot = pilot_company_easiest
-npc_class = lawful, class_light, class_fighter, class_mission, d1, d2
+npc_class = lawful, class_light, class_fighter, class_mission, d1
 
 [NPCShipArch]
-nickname = npc_co_ge_fighter_d3-4
-level = d3
-loadout = co_ge_fighter_loadout_d3-4
+nickname = npc_co_ge_fighter_d2_a
+;;nickname = npc_co_ge_fighter_d3-4
+level = d2
+loadout = co_ge_fighter_loadout_d2_a
 ship_archetype = cv_ship_fighter_verylight01_npc
 state_graph = FIGHTER
-pilot = pilot_company_easiest
-npc_class = lawful, class_light, class_fighter, class_mission, d3, d4
+pilot = pilot_company_easy
+npc_class = lawful, class_light, class_fighter, class_mission, d2
 
 [NPCShipArch]
-nickname = npc_co_ge_fighter_d5-6
-level = d5
-loadout = co_ge_fighter_loadout_d5-6
+nickname = npc_co_ge_fighter_d2_b
+;;nickname = npc_co_ge_fighter_d3-4
+level = d2
+loadout = co_ge_fighter_loadout_d2_b
 ship_archetype = cv_ship_fighter_verylight01_npc
 state_graph = FIGHTER
 pilot = pilot_company_easy
-npc_class = lawful, class_light, class_fighter, class_mission, d5, d6
+npc_class = lawful, class_light, class_fighter, class_mission, d2
 
 [NPCShipArch]
-nickname = npc_co_ge_fighter_d7-8
-level = d7
-loadout = co_ge_fighter_loadout_d7-8
+nickname = npc_co_ge_fighter_d3_a
+;;nickname = npc_co_ge_fighter_d5-6
+level = d3
+loadout = co_ge_fighter_loadout_d3_a
 ship_archetype = cv_ship_fighter_verylight01_npc
 state_graph = FIGHTER
 pilot = pilot_company_med
-npc_class = lawful, class_light, class_fighter, class_mission, d7, d8
+npc_class = lawful, class_light, class_fighter, class_mission, d3
 
 [NPCShipArch]
-nickname = npc_co_ge_fighter_d9-10
-level = d9
-loadout = co_ge_fighter_loadout_d9-10
+nickname = npc_co_ge_fighter_d3_b
+;;nickname = npc_co_ge_fighter_d5-6
+level = d3
+loadout = co_ge_fighter_loadout_d3_b
 ship_archetype = cv_ship_fighter_verylight01_npc
 state_graph = FIGHTER
 pilot = pilot_company_med
-npc_class = lawful, class_light, class_fighter, class_mission, d9, d10
+npc_class = lawful, class_light, class_fighter, class_mission, d3
 
-[NPCShipArch]
-nickname = npc_co_ge_fighter_d11-12
-level = d11
-loadout = co_ge_fighter_loadout_d11-12
-ship_archetype = cv_ship_fighter_verylight01_npc
-state_graph = FIGHTER
-pilot = pilot_company_hard
-npc_class = lawful, class_light, class_fighter, class_mission, d11, d12
+;;[NPCShipArch]

```

Comment:
Are those commented sections needed for something?

Reply:
These are old (not used anymore) entries, i used them to compare with old and new Npc Balancing, i will create an extra branch for "clean up"